client/scripts/views/components/component_kit/icons.ts

```diff
@@ -125,6 +125,10 @@ export const ReplyIcon: m.Component<
             }),
           ]
         ),
+      // size === IconSize.LG &&
```

Comment:
Can we just delete this?

Reply:
I believe this was an @gdjohnson change - can probably be deleted ?

Reply:
let's just delete it